### ISSUE TITLE
(Maybe ) Bug : complication "battery state" show 0% when sentry mode is off

### ISSUE THREAD

**Issue by user858753257** [itsMeDavidV/Watchla-Support#43]
Hi , 

I have turned off sentry mode at my home location via the in car monitor . 

I have done this three days ago . Since than the battery state widget doesn't show content , except when I click on it and it loads data . 

The last week  I had sentry on all
Day and it worked fine . 

Do you have an idea ?

**Comment by itsMeDavidV** [comment]
When you leave Sentry Mode off, the car is more prone to fall into deep sleep. And just like a human, it takes a long time to wake up from deep sleep. For Tesla, it can take anywhere from 15 seconds to minutes, but on average it takes around 20-30 seconds.

The bulk of complications updates (Eg: battery state) happen in the background.
 
Due to the limited capacity of the Apple Watch's battery, WatchOS only affords each app around roughly 15 seconds per background session. And each app is limited on the number of bg sessions they can have per day.

So long story short, WatchOS simply doesn't give enough time for Watchla to wait for your car to wake up and send back an update on how it's doing when your car is in deep sleep. Hopefully Apple expands capabilities for WatchOS in the future so these types of things don't happen :/

**Comment by itsMeDavidV** [comment]
Sometimes when data disruptions happen Watchla reflects last known data, however it does no good to show days old data if your car has been sitting in deep sleep. (Eg: you left your car at the airport for a few days and turned sentry mode off)
